bobtemplates/plone/theme_package/buildout.cfg.bob

```diff
@@ -1,5 +1,5 @@
 [buildout]
-extends = http://dist.plone.org/release/{{{ plone.version }}}/versions.cfg
+extends = http://dist.plone.org/release/{{{ plone.version }}}-latest/versions.cfg
```

Comment:
I'm :+1: with this change. Others might want to pick it explicitly to a specific version, but IMO that would get out of date quickly.